src/azul/drs.py

```diff
@@ -258,6 +261,11 @@ def access_url(url):
 
 class DRSClient:
 
+    def __init__(self, http: urllib3.PoolManager = None):
+        if http is None:
+            http = urllib3.PoolManager(ca_certs=certifi.where())
```

Comment:
This pattern leads to duplication. Every class that needs an http client will end up duplicating the pool manager instantiation. It's better to be honest about the dependency, make the argument required and provide a centralized means of obtaining a client.